src/TEINode.js

```diff
@@ -35,6 +35,24 @@ class TEINode extends React.Component {
     if (!this.props.teiNode) return null
 
     if (this.props.teiNode.nodeType === 3) {
+      if (this.props.availableRoutes.includes('text()')) {
+        const propsClone = {
```

Comment:
perhaps it might be possible to move the following code to its function – the exact duplication a few lines later is maybe a little bit ugly …?